### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by AllenDowney**
Clarifying setup instructions in response to feedback.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -361,16 +361,17 @@ and because it does a user-level install, it will not interfere with other Pytho
 If you have the choice of Python 2 or 3, choose Python 3.
 
 There are two ways to get the libraries you need:
-* You can install them in an existing Conda environment.
-* You can create a new Conda environment.
+* You can install them in an existing Conda environment (not recommended).
```
Comment (abostroem) [inline index.md:364]: Can we flip the order here to have the recommended option first?

Comment (AllenDowney) [reply index.md:364]: Yes, but then we might want to reverse the order of the two subsections that follow.

Comment (abostroem) [reply index.md:364]: Agreed - let's flip the order and add a `(not recommended)` to `#### Install libraries in an existing Conda environment`

Comment (AllenDowney) [reply index.md:364]: Ok.  Please merge this PR and I will open another one with these changes (just to keep the work flow simple).

(Or feel free to make the changes yourself.)

**Pull request merged**